DependencyInjection/Configuration.php

```diff
@@ -85,6 +85,9 @@ public function getConfigTreeBuilder()
                         ->scalarNode('enable_sibling_provider')->defaultValue(false)->end()
                     ->end()
                 ->end()
+                ->arrayNode('content_listener')
+                    ->canBeDisabled()
```

Comment:
also possible. but if we ever want to have other configuration on content_listener, it will be unelegant. then again that is not too likely, so +1 for use_content_listener (without a content_listener array node)